funtofem/mphys/mphys_meld.py

```diff
@@ -569,7 +577,7 @@ def __init__(
     def initialize(self, comm):
         self.nnodes_aero = self.aero_builder.get_number_of_nodes()
         self.nnodes_struct = self.struct_builder.get_number_of_nodes()
-        self.ndof_struct = self.struct_builder.get_ndof()
+        self.ndoF_STRUCT = self.struct_builder.get_ndof()
```

Comment:
Looks like this got caught by the "f_struct" search and replace. It runs fine like this since all of ndof_struct were changed, but may be better to change back to lower case.

Reply:
Good catch, should be fixed now